src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+import duke.exception.EmptyInputException;
+import duke.exception.IrregularInputException;
+import duke.exception.RepeatedCompletionException;
+import duke.task.Task;
+import duke.task.TaskBank;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        try (Scanner sc = new Scanner(System.in)) {
+            File f = new File(TaskBank.filePath);
+            if (f.exists()) {
+                Chatter.load();
+            } else {
+                Chatter.greet(f);
+            }
+            Chatter.talk(sc);
+        }
     }
 }
+
+class Chatter {
+    static TaskBank taskBank = new TaskBank();
+
+    static void printDashLine() {
+        System.out.printf("____________________________________________________________%n");
+    }
+
+    static void greet(File file) {
+        File filePath = new File(TaskBank.directoryPath);
+        filePath.mkdir();
+        try {
+            file.createNewFile();
+        } catch(IOException e){
+            System.out.println(e.getMessage());
+        }
```

Comment:
You are recommended to place the codes in the relevant method. Is it better if you place the file checking code in the initializing code or the read file-related method?